src/types.js

```diff
@@ -319,8 +319,26 @@ export type TabNavigationOptionsPropsType = {|
  * Summary of a PM conversation (either 1:1 or group PMs).
  */
 export type PmConversationData = {|
+  /**
+   * A comma-separated (numerically-)sorted sequence of the IDs of the users
+   * involved in this conversation. Does not includes the self-user iff there
+   * are _exactly_ two recipients.
+   *
+   * (This unusual specification is intended to simultaneously match the
+   * disjoint key-spaces of `unreadPms` and `unreadHuddles`.)
+   */
   ids: string,
+
+  /** The most recent message in this conversation. */
   msgId: number,
+
+  /**
+   * A comma-separated sequence of the emails of the users involved in this
+   * conversation, sorted by user ID. Does not include the self-user, unless the
+   * self-user is the only recipient.
+   */
   recipients: string,
```

Comment:
Is this true?

I don't think it is. See the implementation of `normalizeRecipientsSansMe`, and `normalizeRecipients` which it invokes. We sort there, by email address.

Reply:
Whoops! Will fix.